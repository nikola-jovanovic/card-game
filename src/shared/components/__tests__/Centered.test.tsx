import React from 'react'
import { render } from '@testing-library/react'

import Centered from '../Centered'

describe('Centered', () => {
  it('render', () => {
    const { container } = render(<Centered>text</Centered>)

    expect(container).toMatchSnapshot()
  })
})