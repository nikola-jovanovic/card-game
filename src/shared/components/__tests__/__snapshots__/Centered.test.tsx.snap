// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Centered render 1`] = `
.c0 {
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  height: 100%;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  padding: 10px;
}

<div>
  <div
    class="c0"
  >
    text
  </div>
</div>
`;
